### ISSUE TITLE
 Upgrade node version used by the action from 12 to 16

### ISSUE THREAD

**Issue by troymccabe** [mindsers/changelog-reader-action#74]
Deprecation notice is being thrown. Can this get bumped from 12 to 16?

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: <snip>, mindsers/changelog-reader-action, <snip>
```

**Comment by mindsers** [comment]
Next release should solve this

**Comment by mindsers** [comment]
Should be fixed with 2.2.1
